modules/ROOT/pages/dw-arrays-functions-takewhile.adoc

```diff
@@ -25,7 +29,9 @@ _Introduced in DataWeave 2.2.0. Supported by Mule 4.2 and later._
 === Example
 
 This example iterates over the elements in the array and selects only those
-where in index is `&lt;= 2`. It outputs the result into an array.
+where the index is `&lt;= 2`. It stops selecting elements when it reaches an 
+element that is greater than 2. Notice that it does not select the 3 because 4 
```

Comment:
Can we omit _the_ before 3? >  _Notice that it does not select  3 because 4 _
or  _Notice that it does not select  the number 3 because 4_

Reply:
done